src/table/body-cell/index.tsx

```diff
@@ -16,35 +16,37 @@ const submitHandlerFallback = () => {
 };
 
 interface TableBodyCellProps<ItemType> extends TableTdElementProps {
-  column: TableProps.ColumnDefinition<ItemType>;
+  column: TableProps.ColumnDefinition<any>;
   item: ItemType;
   isEditing: boolean;
   onEditStart: () => void;
   onEditEnd: () => void;
-  submitEdit?: TableProps.SubmitEditFunction<ItemType>;
+  submitEdit?: TableProps.SubmitEditFunction<any>;
   ariaLabels: TableProps['ariaLabels'];
 }
 
-function TableCellEditable<ItemType>({
-  className,
-  item,
-  column,
-  isEditing,
-  onEditStart,
-  onEditEnd,
-  submitEdit,
-  ariaLabels,
-  isVisualRefresh,
-  ...rest
-}: TableBodyCellProps<ItemType>) {
+const TableCellEditable = React.forwardRef(function TableCellEditable<ItemType>(
+  {
+    className,
+    item,
+    column,
+    isEditing,
+    onEditStart,
+    onEditEnd,
+    submitEdit,
+    ariaLabels,
+    isVisualRefresh,
```

Comment:
Since `isVisualRefresh` is a calculated-once constant now, I always remove it from any props when refactoring code (and recommend you to do the same)

Reply:
Thanks for the call-out. Anything I can do to remove one more prop from these components is welcome!